Source/GRTeclynCore/GRAMRLevel.cpp

```diff
@@ -199,6 +199,15 @@ void GRAMRLevel::post_timestep(int /*iteration*/)
 
         FourthOrderInterpFromFineToCoarse(S_crse, 0, NUM_VARS, S_fine, ratio);
     }
+    if (simParams().nan_check)
+    {
+        amrex::MultiFab &S_new = get_new_data(State_Type);
+        if (S_new.contains_nan(0, S_new.nComp(), amrex::IntVect(0), true))
```

Comment:
I think it would be good to add a space before this block and rename `S_new` -> `state_new` (as we will do elsewhere in other PRs).

The following suggestion is almost certainly not clang-formatted correctly but you get the idea:

```suggestion

    if (simParams().nan_check)
    {
        amrex::MultiFab &state_new = get_new_data(State_Type);
        if (state_new.contains_nan(0, state_new.nComp(), amrex::IntVect(0), true))
```